tachyon/math/polynomials/multivariate/multilinear_extension_ops.h

```diff
@@ -16,6 +16,19 @@ class MultilinearExtensionOp<MultilinearDenseEvaluations<F, MaxDegree>> {
  public:
   using D = MultilinearDenseEvaluations<F, MaxDegree>;
 
+  static MultilinearExtension<D> Add(const MultilinearExtension<D>& self,
```

Comment:
How about adding an early return here as well when self or other is zero?

Reply:
`IsZero()` checking possible is expensive, but I think `CHECK_EQ(l_evaluations.size(), r_evaluations.size());` seems a bug! I'll fix it!